lib/impress.constants.js

```diff
@@ -21,7 +21,7 @@ impress.FILE_PARSE_ERROR = 13;
 impress.DIR_EXISTS = 50;
 impress.DIR_NOT_EXISTS = 51;
 
-impress.USE_STRICT = '"use strict";\n';
+impress.USE_STRICT = '\'use strict\';\n';
```

Comment:
Unrelated style change, but I'm fine with it being in this PR. I'd prefer it to be a second commit nevertheless, though.